packages/protocol/contracts/common/Accounts.sol

```diff
@@ -334,7 +334,9 @@ contract Accounts is
 
   /**
    * @notice Gets validator payment delegation settings.
-   * @return Beneficiary address and fraction of payment delegated.
+   * @param address of the validator.
+   * @return Beneficiary address.
+   * @return 24 decimal fraction of payment delegated.
```

Comment:
For now, let's keep a single `@return` like in the rest of the codebase for consistency. I added an issue to fix this throughout the codebase: #9268.